modules/meeting/app/controllers/projects/meetings_controller.rb

```diff
@@ -0,0 +1,109 @@
+#-- copyright
+# OpenProject is an open source project management software.
+# Copyright (C) 2012-2023 the OpenProject GmbH
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See COPYRIGHT and LICENSE files for more details.
+#++
+
+class Projects::MeetingsController < BaseController
+  before_action :find_project, only: %i[index new create]
+  before_action :build_meeting, only: %i[new create]
+  before_action :convert_params, only: %i[create update]
+  before_action :authorize
+
+  def index
+    @meetings = @project.meetings
+  end
+
+  def create
```

Comment:
The authorization check for projects is missing from create. By altering the meeting form, I am able to create a meeting in a project where I don't have access to.
![Jun-15-2023 13-34-53](https://github.com/opf/openproject/assets/83396/0f750db8-9378-4762-8b3e-d917bc39c8fe)